kodilla-spring-basic/src/main/java/com/kodilla/spring/basic/spring_configuration/homework/CarFactory.java

```diff
@@ -0,0 +1,36 @@
+package com.kodilla.spring.basic.spring_configuration.homework;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+
+@Configuration
+public class CarFactory {
+
+    @Bean
+    public SUV createSUV() {
+        return new SUV();
+    }
+
+    @Bean
+    public Car carSelection() {
+        Car car;
+        Random generator = new Random();
+        List<String> seasons = Arrays.asList("Winter", "Spring", "Summer", "Autumn");
+        int select = generator.nextInt(3);
```

Comment:
Co ta metida robi? Nie ma pojedynczej odpowiedzialnosci.
To nie jest rozwiazanie zadania